static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -70,10 +72,49 @@
     }
 }
 
+.adverts--tone-supported {
+    background: none;
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        border-top: 1px solid $news-main-2;
+
+        @include mq(tablet) {
+            left: calc(50% - 370px);
```

Comment:
Just keep in mind that calc() is not supported on opera mini and some older android versions. Opera mini seems to have almost 5% global usage. If anything I would check with QA to get the proper number as https://frontend.gutools.co.uk/analytics/quality shows only up to 600k visitors.